<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string>Sources</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Kafka</string>
         </property>
         <item>
          <property name="text">
           <string>New Subitem</string>
          </property>
         </item>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="connectionData"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHigway">
    <property name="title">
     <string>Higway</string>
    </property>
    <addaction name="actionAdd_connection"/>
   </widget>
   <addaction name="menuHigway"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionjyg">
   <property name="text">
    <string>jyg</string>
   </property>
  </action>
  <action name="actionjiuyiuyiuy">
   <property name="text">
    <string>jiuyiuyiuy</string>
   </property>
  </action>
  <action name="actionAdd_connection">
   <property name="text">
    <string>Add connection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
